atlasdb-config/src/main/java/com/palantir/atlasdb/http/UserAgents.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright 2017 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.http;
+
+import java.util.Optional;
+
+import com.google.common.annotations.VisibleForTesting;
+
+public final class UserAgents {
+    private static final String ATLASDB_CLIENT = "atlasdb";
+
+    public static final String DEFAULT_VALUE = "unknown";
+    public static final String USER_AGENT_FORMAT = "%s-" + ATLASDB_CLIENT + " (%s)";
```

Comment:
If we end up including the separate AtlasDB version, we'll need to adjust some log parsing